.github/workflows/build.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           node-version: 20
 
       - name: Install dependencies
-        run: npm ci --omit=dev
+        run: npm ci
```

Comment:
Why do we need dev dependencies in build ?

Reply:
Actually, next js build runs typescript check before build. And, some packages have their typings included in the dev dependencies, causing the build job to fail prematurely because of typescript compliation error.